src/librustc_lint/lib.rs

```diff
@@ -286,6 +287,11 @@ pub fn register_builtins(store: &mut lint::LintStore, sess: Option<&Session>) {
             reference: "issue TBD",
             edition: Some(Edition::Edition2018),
         },
+        FutureIncompatibleInfo {
```

Comment:
I think we should not have the lint in this list. It is not a future incompatibility lint, at least not yet. =)